profile.yaml

```
@@ -0,0 +1,249 @@
+openapi: 3.0.0
+info:
+  title: Simple REST APIs
+  description: This is profile API.
```

Since this is the Platform API definition, I think it needs to be more generic. I'm adding a suggestion below: 
```suggestion
  description: API documentation for the Rokwire Platform.
```
We can add more text to this description defining the platform as we go.

---

Thanks for the proof reading. All points are very valid.